### ISSUE TITLE
Questions About Reproducing CogVideoX-2B ControlNet Results

### ISSUE THREAD

**Issue by yangzhou24** [snap-research/ac3d#26]
Hi,

Thank you for sharing this great project! I have two questions regarding the CogVideoX-2B ControlNet pre-trained weights and training process:

1. Were the provided pre-trained weights strictly trained using the parameters in `ac3d/scripts/train_2b.sh` and `ac3d/training/train_controlnet.py`?
2. When I trained CogVideoX-2B ControlNet on the re10k dataset following the provided code, I noticed that the loss function oscillates during training. Is this behavior expected? In your training process, did the loss converge? If so, what was the approximate final loss value when achieving reasonable results?

Thank you for your time and help!

**Comment by sherwinbahmani** [comment]
Hi,

1. Yes, the provided model is trained with this exact code base and configs.
2. I do not have any loss curve available. You should be able to see the model converging with the camera control after 1 day using 8 GPUs. I mainly looked at the visuals matching the input trajectory to judge convergence.

**Comment by yangzhou24** [comment]
Thank you for your timely help!

**Comment by yangzhou24** [comment]
I noticed that in the implementation of the ControlNet module, the code does not  set the `cfg.init_from_transformer` and not load `CogVideo weights`, which seems different from the typical usage of ControlNet. Could you please clarify the reasoning behind this design choice?
